src/app/containers/Headings/index.jsx

```diff
@@ -38,7 +39,13 @@ const HeadingsContainer = ({ blocks, type }) => {
 
   return (
     <GridConstrain>
-      <Heading text={text}>{renderText()}</Heading>
+      <ServiceContextConsumer>
```

Comment:
Bit annoying about the enzyme issue meaning we cant use the hook here, can you make an issue to replace it/the tests please?

Reply:
Correct, I've mentioned in the PR description this needs to be addressed.
>Headings container purposefully does not use useContext because it has enzyme tests which do not yet support useContext this will be resolved in #1569